metrics/README.md

```diff
@@ -25,3 +27,33 @@ the name of the metric. The second one contains the value (float or integer).
 The third one contains the description of the metric (in general, NOT
 for this particular file). A space is mandatory between the first and the
 second column, and between the second and the third columns.
+
+## Metrics in make
+
+Metrics could be calculated in **measure** step presented in makefile.
```

Comment:
@IlnurHA try to avoid bold and italic in tech documents, see: https://www.yegor256.com/2019/05/21/dont-improvise.html